app/src/ui/app.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react'
 import * as crypto from 'crypto'
+import { escape } from 'querystring'
```

Comment:
The `querystring` module is marked as legacy in the Node docs. Could you please use [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) instead? It won't have any practical effect but that's what we're using elsewhere and it's well documented.

Reply:
Nice catch @niik! I changed both instances of `escape` to `encodeURIComponent`. 

In the original PR, I used the encoding in `_openCreatePullRequestInBrowser` as a reference to use `escape` in the first place.